### PR TITLE
Reformat CUAHSI Results

### PR DESCRIPTION
**Pull request by rajadain**
## Overview

Using the guidelines from https://github.com/WikiWatershed/model-my-watershed/issues/1931#issuecomment-306838234, add aggregate fields to the record grouping, include them in the CUAHSI resource model and serializers, and show them in the front end as requested in https://github.com/WikiWatershed/model-my-watershed/issues/1931#issuecomment-311170676. Also add rudimentary scrolling to ease testing.

Builds on top of #2048 
Connects #1990 
Connects #1991 

### Demo

image

### Notes

In some cases, the Sample Medium has a value `Unknown`. Should these be kept in the UI (as in the screenshot above) or omitted? /cc @aufdenkampe @emiliom 

I also went with semicolons for delimiting a list with commas in the values, which felt more standard to me, but if we'd rather have the hyphens I can easily change to that.

I added some rudimentary scrolling to assist in testing, but actual scrolling should be added as part of #1938.

## Testing Instructions

 * Check out the branch and `bundle`
 * Go to [:8000/?bigcz](http://localhost:8000/?bigcz)
 * Go to the search page and search for `water`
 * Inspect CUAHSI / WDC results and ensure that they match the formatting as requested in #1990 
 * Ensure that only the `NWISDV` values have links in the title

### CODE REVIEW

**Comment by rajadain** [comment]
Just added a commit that implements #1991, which was simple since we don't have to deal with pagination for CUAHSI.

**Comment by rajadain** [comment]
Updated PR to point to `develop`

File: src/mmw/apps/bigcz/clients/cuahsi/search.py [hunk 1]
```diff
@@ -102,25 +111,38 @@ def parse_records(series, services):
...
+    records = []
+    for location, group in groups.iteritems():
+        records.append({
+            'ServCode': group[0]['ServCode'],
+            'ServURL': group[0]['ServURL'],
+            'location': group[0]['location'],
+            'Sitename': group[0]['Sitename'],
+            'latitude': group[0]['latitude'],
+            'longitude': group[0]['longitude'],
+            'sample_mediums': ', '.join(sorted(set([r['samplemedium']
```
Comment (arottersman) [inline src/mmw/apps/bigcz/clients/cuahsi/search.py]: If we're going to normalize the aggregate attribute names into snakey_case, we could consider normalizing all the attribute names

File: src/mmw/js/src/data_catalog/templates/cuahsiSearchResult.html [hunk 2]
```diff
@@ -1,19 +1,26 @@
...
-    </div>
-{% endif %}
-{% if author %}
-    <div class="resource-author">
-        {{ author }}
-    </div>
-{% endif %}
-<div class="resource-test-field">
-    {{ test_field }}
+<div class="resource-concept_keyword">
```
Comment (arottersman) [inline src/mmw/js/src/data_catalog/templates/cuahsiSearchResult.html]: Consider removing the class names.

**Comment by rajadain** [comment]
Thanks for taking a look. Squashed fixups, merging as soon as CI is green.

**Pull request merged**